### DISCUSSION TITLE
Барьер

### DISCUSSION THREAD

**Discussion by Vladislav894** [Sokol-Games-GmbH/PixelCombats.PublicApi#193] [Ideas]
Здравствуйте! Я бы хотел вам предложить такую идею как "барьер" это будет прозрачный, но не уничтожаемый блок. Чтобы его увидеть надо будет взять его в руки. Думаю он поможет строителям в создании карт. Т.к есть некоторые места, где не хочется, чтобы туда попадали игроки

**Comment by kkohno** [comment]
понял, обдумаем насчет барьеров
